plugins/adrv9002.c

```diff
@@ -927,6 +1189,1009 @@ static void load_profile(GtkFileChooser *chooser, gpointer data)
 		gtk_file_chooser_set_filename(chooser, "(None)");
 }
 
+static  void profile_gen_set_debug_info(gpointer data, char *string)
+{
+	struct plugin_private *priv = data;
+	gtk_label_set_text(GTK_LABEL(gtk_builder_get_object(priv->builder, "label_profile_debug")), string);
+}
+
+static void profile_gen_save_type_changed(GtkComboBoxText* self, struct plugin_private *priv)
+{
+	// 0 - stream image
+	// 1 - profile
+	bool save_type = atoi(gtk_combo_box_get_active_id(GTK_COMBO_BOX(self)));
+
+	char filename[64];
```

Comment:
I'm very biased by linux kernel coding style where all the declarations are in the beginning of the function. I know that's arguable but since you're working on an existing plugin, consistency matters. So, again, please be consistent with the rest of the code in the plugin :)